customer_satisfaction/streamlit_app.py

```diff
@@ -0,0 +1,96 @@
+import streamlit as st
```

Comment:
I think the presentation of this probably still needs a bit more work IMO. I'd take outside opinions, too. @htahir1 @AlexejPenner the demo app is [here](https://share.streamlit.io/ayush714/customer-satisfaction/main)

Reply:
So, from my perspective, I think:- 

- We can add batch prediction there. 
- We can add something like someone can select the model and start training. 
- Adding a brief about how the model is working?

Reply:
Is it possible to prefill the fields instead of having all of them be 0 ?

Reply:
I don't know if we have such features of autofill in streamlit, I have to check

Reply:
@AlexejPenner  I think it has a feature but I only saw for textual input. may be i can implement some custom stuff for this feature.

Reply:
If its not easily doable in a few minutes lets skip this